frontend/src/app/modules/global_search/global-search-input.component.html

```diff
@@ -8,20 +8,39 @@
         <div class="block-5"></div>
       </div>
     </div>
-    <input #inputEl
-           type="text"
-           name="q"
-           id="q"
-           size="20"
-           class="top-menu-search--input"
-           [class.-expanded]="expanded"
-           placeholder="{{text.search}}"
-           autocomplete="off"
-           [ngModel]="searchTerm"
-           (focus) = "resize()"
-           (focusout) = "resize()"
-           (keyup.enter)="submitNonEmptySearch()"
-           (keydown.escape)="inputEl.blur()">
+     <ng-select [items]="results"
+                name="q"
+                id="q"
+                [ngClass]="'top-menu-search--input'"
+                [class.-expanded]="expanded"
+                placeholder="{{text.search}}"
+                bindLabel="subject"
+                (focus)="onFocus()"
+                (focusout)="onFocusOut()"
+                (search)="toggleMenu($event)"
+                (open)="toggleMenu(searchTerm)"
```

Comment:
Was this your choice that once the parameter for `toggleMenu` is `$event` and once `searchTerm`? Looks somehow random but I haven't investigated. If it does not matter please use the same parameter to avoid confusion.